planet/clients/data.py

```diff
@@ -175,11 +175,12 @@ async def create_search(self,
         url = self._searches_url()
 
         # TODO: validate item_types
-        request_json = {
+        request_json: typing.Dict[str, typing.Any] = {
```

Comment:
Without this hint, mypy diagnoses request_json's type as `Dict[str, Collection[Any]]` because name, search_filter, and item_types are all collections, and then complains when we try to add a bool item, which is not a collection.

Reply:
ahh interesting! good catch. I wonder if we want to make this a custom type, eg `JSONType` and move it over, along with `AuthType` to a new `types` module?